php/docker/templates/nginx-conf.d/95-metrics.conf

```
@@ -0,0 +1,16 @@
+server {
+    listen 9145;
+
+    # TODO handle authorization!
+    # allow 192.168.0.0/16;
+    # deny all;
+
+    location /metrics {
+        content_by_lua_block {
+            metric_connections:set(ngx.var.connections_reading, {"reading"})
```

`site_name` and `pod`
```suggestion
            metric_connections:set(ngx.var.connections_reading, {
                "{{ default "-" .Env.STACK_SITE_NAME }}",
                 ngx.var.server_name,
                "reading"
            })
```